src/Api/Managers/NotificationManager.cs

```diff
@@ -0,0 +1,65 @@
+﻿using BLL.Managers;
+using BOL.Data;
+using BOL.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BLL.Managers
+{
+    public class NotificationManager : BaseManager
+    {
+        public NotificationManager(CmsDbContext dbContext) : base(dbContext) { }
+
+        public virtual IEnumerable<Notification> CurrentUser(User userId)
+        {
+            return (from n in dbContext.Notifications
```

Comment:
this userId is already an instance of the current user, so userId.NotificationsForUser() will return already all notifications for this user.
In addition you should order them by time.